scripts/Auth_scripts/main.ps1

```diff
@@ -0,0 +1,39 @@
+# Requires PSVersion 5.1.18362.1171
+. ".\auth.ps1"
+. ".\getpet.ps1"
+. ".\menu.ps1"
+
+Set-StrictMode -version 1.0

```

Comment:
Your just above comment says that this script requires 5.1 version, why do we use strict mode v1.0?

Reply:
In your opinion what version of mode we should use? Also starting from version 2  it "Prohibit references to non-existent properties of an object." which exist in our case if user provide bad URL endpoint and the following error message becomes misleading. Please be so king to elaborate why do we even need to use Set-StrictMode.

Reply:
It just does not make so much sense looking throw the script and almost in the same place there are comment and command line stating for two different things. Please, do not take comments too personal, it is just misunderstanding. Can you explain what your comment just right above states for? And what the difference between strict mode v1.0?
And now answer to your question, the cmdlet Strict-Mode command generates an error when the content of a script violates best-practices coding rules.

Reply:
5.1 version stands for for PowerShell itself. As for strict mode there is 3 versions. In version 2 and 3 there is problem with  properties of an object that's non-existent cause if client doesn't provide proper URL the error message will say that "The property 'StatusCode' cannot be found on this object. Verify that the property exists." but in version 1 It'll say "The remote name could not be resolved". I guess in the end we can use strict mode 2 and 3, but then I'll need somehow handle such errors.

Reply:
Update: everything is good now we are using Strict-Mode v.3 and PowerShell v.5.1!